android/README.md

```diff
@@ -0,0 +1,16 @@
+VkRunner for Android
+===========
+
+This Android part code is based on Vulkan Android tutorial provided by Google.
+Since the tutorial is under Apache 2.0. license, this part of VkRunner also
+follows the license.
+
```

Comment:
Please add build instructions.

If I have guessed correctly:

1. Make sure you have the Android NDK.  (Which version works?  r15 or later I guess...)
2. Set environment variable ANDROID_NDK to point to the NDK (give an example)
3. Run ./gradlew  
4. You should have  (what APK name) where.
5. Install the APK onto the device. (What adb command?)
6. Run the APK.

Is that right?  I don't know what "gradle installDebug" does, but maybe it replaces steps 3-5?
What does "Connect to you Android device" mean?  Is it more than:
- a.  Ensure developer options are turned on
- b. connect the device via USB
- c. Verify you are connected, e.g. by doing "adb logcat"